package.json

```diff
@@ -130,6 +130,7 @@
     "@types/webpack": "^5.28.0",
     "array-to-table": "^1.0.1",
     "command-exists": "^1.2.9",
+    "concurrently": "^9.1.2",
```

Comment:
Skip adding this one, you should only need to add it to the Canon package

Reply:
Done